devicemaps.json

```diff
@@ -267,6 +267,129 @@
             }
           }
         ]
+      },
+      "90500-0151-G13": {
+        "description": "Madrid - 8 ethernet, 1 lte",
```

Comment:
Can you add a little note here like `only difference to the G53 is the last two pci port numbers are switched` something like that